dev/com.ibm.ws.concurrent.mp.1.0/bnd.bnd

```diff
@@ -14,46 +14,37 @@ bVersion=1.0
 javac.source: 1.8
 javac.target: 1.8
 
-Bundle-Name: MicroProfile Context Propagation 1.0 implementation
+Bundle-Name: MicroProfile Context Propagation 1.0 Concurrency function
 Bundle-SymbolicName: com.ibm.ws.concurrent.mp.1.0
-Bundle-Description: Implementation of MicroProfile Context Propagation 1.0; version=${bVersion}
+Bundle-Description: Concurrency function used by MicroProfile Context Propagation 1.0 Implementation; version=${bVersion}
 
 Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
 
 WS-TraceGroup: concurrent
 
 Import-Package:\
   javax.inject;resolution:=optional,\
-  !org.eclipse.microprofile.config,\
-  !org.eclipse.microprofile.config.spi,\
   *
 
-DynamicImport-Package:\
-  org.eclipse.microprofile.config,\
-  org.eclipse.microprofile.config.spi
-
-Private-Package:\
-  com.ibm.ws.concurrent.mp.*
-    
 Export-Package: \
-  com.ibm.ws.concurrent.mp.spi.*
+  !com.ibm.ws.concurrent.mp.resources, \
+  com.ibm.ws.concurrent.mp.*
```

Comment:
I have made the classes that can be non public back to default protected.  I am not sure how to export specific classes.  Maybe I need to make another package to be exported.  If I do that then I will need to do some funny things like have an empty classes that just extends the one that uses the default protected classes, or I need to make the classes public again, but just not export the package so that no one can see the classes outside of the bundle.  Will talk to you about it more tomorrow.